### PR TITLE
Feature/create menu

### PR DESCRIPTION
**Pull request by gabrieldrs**
This commit implements the POST `menus` route.

You can test it by running (make sure the server is running at localhost:8080):

```js
var fs = require("fs");
var request = require("request");

var options = { 
    method: 'POST',
    url: 'http://localhost:8080/api/v1/menus',
    headers: { 'content-type': 'multipart/form-data' },
    formData: {
        file: {
            value: fs.createReadStream('/path/to/file.pdf'),
            options: {
                filename: '/path/to/file.pdf'
            }
        },
        restaurant_name: 'Restaurant Name',
        restaurant_address: 'Restaurant Address',
        restaurant_city: 'Restaurant City'
    }
};

request(options, function (error, response, body) {
    if (error) throw new Error(error);

    console.log(body);
});

```

### CODE REVIEW

**Comment by gabrieldrs** [comment]
Please note that this commits depends on #14

File: api/lib/MulterConfig.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        fileFilter: (req, file, cb) => {
+            if (file.mimetype !== 'application/pdf'){
+                let err = new Error(`Wrong file type: "${file.mimetype}"`);
+                err.status = httpStatus.BAD_REQUEST;
+                return cb(err, false);
+            }
+            return cb(null, true);
+        },
+        limits: {
+            fileSize: 2097152, // 2MB
```
Comment (gportog) [inline api/lib/MulterConfig.js]: @gabrieldrs I don't know if is an issue or not, but I saw that 40% of our sample pass than 2MB and it's directly related with their quality... Do you think is better increase the file Size limit?

Comment (gabrieldrs) [reply api/lib/MulterConfig.js]: Sure, by how much do you think we should limit it?

Comment (gabrieldrs) [reply api/lib/MulterConfig.js]: I've just checked and the largest one that we have is "ks-prime-sushi-bar" with 11MB.

I do think 11MB is too much for a single file in our case, so I will limit to 10MB, and I will consider this specific menu as an outlier.

**Comment by gabrieldrs** [comment]
Updated file size limit to 10MB

**Pull request merged**